src/main/java/core/basesyntax/service/operation/OperationBHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.FruitShop;
+
+public class OperationBHandler implements OperationHandler {
```

Comment:
```suggestion
public class OperationBHandler implements OperationHandler {
```
Я б юзав наступний неймінг: SupplyHandler, ReturnHandler, etc. Щоб навіть розробнику який не писав цей код було зрозуміло, що означає P B S